### ISSUE TITLE
"host:player af toggle lavfi=[dynaudnorm=f=100]" makes video sped up and audio starting from approx 20 sec in

### ISSUE THREAD

**Issue by atrus05** [plexinc/plex-media-player#499]
So I have been using this amazing command in PMP for a good year or so:
```"Ctrl\\+D": "host:player af toggle lavfi=[dynaudnorm=f=100]",```

But since early December (or even before that) it has created an issue where the video is sped up a lot when I start up a video with that command already being activated. It does not stop being sped up until it has caught up with the audio which started about 20 seconds into the video.

Platform OSX 10.11.6
PMP version: Has been happening for at least a month and a half with all the Ninja versions.

**Comment by wm4** [comment]
Yeah, the huge buffering messes up audio position tracking.

**Comment by wm4** [comment]
Hm, took a look at it again, and can't reproduce it. Maybe we just need to update deps...

**Comment by atrus05** [comment]
I can still reproduce it. Even after updating to OSX Sierra.

**Comment by wm4** [comment]
Still can't reproduce. This time I actually tried in on OSX. I only noticed that if you don't release the cmd key, you get autorepeat and "weird" behavior. Other than that seems to be fine.

**Comment by atrus05** [comment]
@wm4 I get it on both my machines, so I just want to be sure we are doing the exact same thing:
STR:
1. Open PMP and start playing a video. Whichever video.
2. Trigger the command then exit the video, but do not exit PMP.
3. Start playing Veronica Mars S01E01 from my server.

Video should now be sped up for about 10 seconds. By a lot.

**Comment by wm4** [comment]
Still can't reproduce with exactly those steps and on OSX (though I plyed the file from my local server). Tried build 566.

**Comment by atrus05** [comment]
@wm4 A fellow Ninja was not able to reproduce this issue on my Veronica Mars s01e01 episode. But he then switched the quality to 4Mbps, 720p. And voila. The issue started happening for him.

His debug overlay screenshots:
With the command enabled (and issue happening):
![pasted image at 2017_03_01 12_39 am](https://cloud.githubusercontent.com/assets/840936/23527049/a8ab21d6-ff94-11e6-99f9-8568288ffae0.png)

Without DRC (the issue not happening):
![pasted image at 2017_03_01 12_40 am](https://cloud.githubusercontent.com/assets/840936/23527071/bd1d2d8a-ff94-11e6-899d-a42bb35e7f36.png)

I have also tested bypassing my receiver to see if that could help anything for some reason. But that did not help. HDMI straight from my Mac Mini to my Plasma TV causes the exact same issue.

So far I have only been able to reproduce this with video files that have AAC as audio track.
Videos with FLAC, AC3 and DTS (DCA) does not seem to cause this bug to show up.

**Comment by wm4** [comment]
Yes, with transcoding this will probably happen, because then mpv can't seek to restore the audio position.

**Comment by atrus05** [comment]
This is still relevant in Mac OS X 10.13.2 using 2.8.0.813, and should not be closed.

**Comment by atrus05** [comment]
Still relevant in Mojave using PMP .924 and earlier.
